packages/edit-site/src/components/header/navigation-toggle/index.js

```diff
@@ -31,10 +31,19 @@ function NavigationToggle( { icon, isOpen, onClick } ) {
 		};
 	}, [] );
 
+	const { setNavigationPanelActiveMenu } = useDispatch( 'core/edit-site' );
+
 	if ( ! isActive ) {
 		return null;
 	}
 
+	const handleClick = ( event ) => {
+		if ( isOpen ) {
+			setNavigationPanelActiveMenu();
```

Comment:
`Non-blocking`:

Definitely nitpicky, but I would prefer invoking `setNavigationPanelActiveMenu` with "root" because it is more explicit. Curious to hear your thoughts though.

Reply:
My reasoning would be that `root` is part of the component's logic, and it shouldn't be a concern of the consumer.

On the other hand, the consumer still needs to know of that keyword to create nested menus, so yeah I guess it's not really an essential change.
I'll update to add `root` back into the navigation panel.